packages/react-sdk/docusaurus/docs/React/01-basics/03-quickstart.mdx

```diff
@@ -24,58 +25,35 @@ const userId = 'user-id';
 const token = 'authentication-token';
 const user: User = { id: userId };
 
-export default function App() {
-  const [client, setClient] = useState<StreamVideoClient>();
-  const [call, setCall] = useState<Call>();
-
-  useEffect(() => {
-    const myClient = new StreamVideoClient({ apiKey, user, token });
-    setClient(myClient);
-    return () => {
-      myClient.disconnectUser();
-      setClient(undefined);
-    };
-  }, []);
-
-  useEffect(() => {
-    if (!client) return;
-    const myCall = client.call('default', 'my-first-call');
-    myCall.join({ create: true }).catch((err) => {
-      console.error(`Failed to join the call`, err);
-    });
-
-    setCall(myCall);
-
-    return () => {
-      setCall(undefined);
-      myCall.leave().catch((err) => {
-        console.error(`Failed to leave the call`, err);
-      });
-    };
-  }, [client]);
-
-  if (!client || !call) return null;
+const client = new StreamVideoClient({ apiKey, user, token });
+const call = client.call('default', 'my-first-call');
+myCall.join({ create: true });
```

Comment:
```suggestion
await myCall.join({ create: true });
```

Reply:
I'm not sure if top-level await is supported in Vite and React. That's why I left it out.